learning_resources/management/commands/update_topics.py

```diff
@@ -1,42 +1,68 @@
-"""Management command for populating LearningResourceTopic data"""
+"""
+Management command for populating LearningResourceTopic data, either from the
+default location or from an update file.
+
+The topics file data should be in the format described in the README-topics.md
+in the data folder.
+
+Use this for testing - if you need to push out changes to the data, roll the
+change into a migration in the data_fixtures app.
+"""
+
+from pathlib import Path
 
 from django.core.management import BaseCommand
 
 from learning_resources.models import LearningResourceTopic
-from learning_resources.utils import upsert_topic_data
+from learning_resources.utils import dump_topics_to_yaml, upsert_topic_data_file
 from main.utils import now_in_utc
 
 
 class Command(BaseCommand):
     """Update LearningResourceTopic data"""
 
-    help = "Update LearningResourceTopic data"
+    help = (
+        "Update LearningResourceTopic data from a yaml file. Optionally "
+        "dump the updated topics to a file."
+    )
```

Comment:
I think this might be better left to the data_fixtures app, @shanbady can you chime in?

Also, in terms of the option for generating the yaml file, seems like there might be an issue if the current data in the db is incomplete?  Should the yaml file only be generated from a presumably 100% complete CSV file ?

Reply:
Yea thats probably the way to go. I feel like anything static fixture related should be consolidated under the data fixtures app

Reply:
I removed it. The upsert APIs can be run from a shell if someone needs to process a file or string manually for whatever reason.

The upsert functions work fine with deltas, so the incoming data doesn't need to be complete.